smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/httpResponse/HTTPResponseBindingErrorGenerator.kt

```diff
@@ -78,6 +78,7 @@ class HTTPResponseBindingErrorGenerator(
 
         ctx.delegator.useShapeWriter(httpBindingSymbol) { writer ->
             writer.addImport(SwiftDependency.CLIENT_RUNTIME.target)
+            writer.addImport(unknownServiceErrorSymbol)
             writer.addImport(customizations.baseErrorSymbol.namespace)
```

Comment:
Now that `UnknownAWSHTTPServiceError` in AWSClientRuntime is marked with `@_spi`, this import needs to be added. But don't worry, anyone that only uses smithy-swift won't see the difference because `addImport` takes care of duplicate imports.